emailbomber.py

```diff
@@ -14,8 +14,12 @@
 print("If you would like to spam more than one email, separate the emails by commas (example@gmail.com, example2@hotmail.com, example3@myspace.com)") #Tells user how to email-bomb more than one email.
 receiver = input("Specify the email(s) you would like to email-bomb -> ") #Enter the email(s) that you would like to email-bomb.
 message = input("Enter your email-bomber message -> ") #The message that the email user(s) will receive.
-count = int(input("Enter a number for the amount of emails to be sent -> ")) #The amount of emails to be sent to the receiver(s).
-
+try:
+    count = int(input("Enter a number for the amount of emails to be sent -> ")) #The amount of emails to be sent to the receiver(s).
+except ValueError:
+    print("Please enter a number for the amount of emails to be sent.")
+    input("Press enter to exit.")
+    quit()
```

Comment:
While trying to get a valid input from the user, you should put the try-catch-block in an infinite loop, otherwise its not that user-friendly